pkgs/development/tools/rubocop/default.nix

```diff
@@ -0,0 +1,20 @@
+{ stdenv, lib, bundlerEnv, ruby, makeWrapper }:
+
+bundlerEnv rec {
+  name = "rubocop-${version}";
+  version = "0.47.0";
+
+  inherit ruby;
+
+  gemfile = ./Gemfile;
+  lockfile = ./Gemfile.lock;
+  gemset = ./gemset.nix;
```

Comment:
There is a new option called `gemdir = ./.;` that can be used to automatically infer all these files.